blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -51,6 +51,10 @@ None
 Fixes
 =====
 
+- Corrected the repository documentation: the default value of the ``fs``
```

Comment:
We build the docs on the release branches, not on tagged releases, so I think it's not necessary to have a changes entry for this. 

(cc: @seut  what do you think?)

Reply:
Agree we don't add changes for docu fixes.